scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -125,22 +133,32 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
 	                  <td>
 	                  {% if ind.phenotype==2 %} <!--for later use-->
 	                    {% do affected.append(ind.display_name) %}
-	                    <span class="badge badge-pill badge-danger" title="Affected">YES</span>
-	                  {% else %}
-	                    <span class="badge badge-pill badge-secondary" title="Not affected">NO</span>
+                      {% if cancer %}
+                        <span class="badge badge-pill badge-danger" title="{{ ind.phenotype_human }}">{{ ind.phenotype_human }}</span>
+                      {% else %}
+                        <span class="badge badge-pill badge-danger" title="Affected">YES</span>
+                      {% endif %}
+                  {% else %}
+                      {% if cancer %}
```

Comment:
Here also, I suggest to treat RD and cancer in the same way by using ind.phenotype_human